cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -123,6 +123,15 @@
     public static final String MANAGE_COPROCESSORS = "master.manage.hbase.coprocessors";
     public static final String CLIENT_RETRIES = "hbase.client.retries.number";
     public static final String RPC_TIMEOUT = "hbase.rpc.timeout";
+    /** Whether to use latest HBase version when HBase version is unknown. cdap_set_hbase() method
+     * in cdap-common/bin/functions.sh must also be updated if this String is changed */
+    public static final String HBASE_VERSION_FOR_UNKNOWN_VERSION = "hbase.compat.module.version";
```

Comment:
the value for `hbase.compat.module.version` are actually the following two constants ""auto.exact" or "auto.latest". As per Terence, this setting may have more options rather than a boolean